views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -83,17 +178,47 @@ Cypress.Commands.add('publishTest', (testName) => {
         cy.contains(testName).click({ force: true });
     });
 
-    // Publish example test
+    cy.wait('@editTest');
+
+    // Publish selected test
     cy.get(setupSelectors.testsPage.testPublishButton).click();
 
+    if (deliveryType === 'remote') {
+        // Selects TAO Remote tab
+        cy.get(setupSelectors.testsPage.deliveryTypeTabs).contains('TAO Remote').click();
+    }
+
+    // Select suitable tenant
+    cy.get('.deliver-tenant-list').select('3');
```

Comment:
It should happen only when I would like to publish to remote.

Reply:
Now wrapped in `if`.